mysql-test/columnstore/basic/t/MCOL-5572-autoincrement-filtering.test

```diff
@@ -0,0 +1,22 @@
+--source ../include/have_columnstore.inc
+
+SET default_storage_engine=columnstore;
+
+--disable_warnings
+DROP DATABASE IF EXISTS mcol5572;
+--enable_warnings
+
+
+CREATE DATABASE mcol5572;
+USE mcol5572;
+
+create table foo (col1 int) engine=columnstore;
+insert into foo values ( 1 ), ( 2 ), ( 3 ), ( 4 ), ( 5 );
+Alter table foo add column newcol bigint comment 'autoincrement';
+select callastinsertid('foo');
+select tablename, autoincrement, nextvalue from calpontsys.syscolumn  where autoincrement = 'y';
+select tablename, autoincrement, nextvalue from calpontsys.syscolumn  where autoincrement != 'n';
+select tablename, autoincrement, nextvalue from calpontsys.syscolumn  where autoincrement = 'n';
```

Comment:
Can you please include the `schema` and `tablename` in the WHERE clause of the above 3 SELECT queries on `calpontsys.syscolumn`. This is because if this test is run on an existing installation with other databases and tables, it will return more rows than expected and the test will fail. Here is the updated statements (please note the usage of back tick ` around the schema keyword which is required):
```sql
select tablename, autoincrement, nextvalue from calpontsys.syscolumn  where autoincrement = 'y' and `schema`='mcol5572' and tablename='foo';
select tablename, autoincrement, nextvalue from calpontsys.syscolumn  where autoincrement != 'n' and `schema`='mcol5572' and tablename='foo';
select tablename, autoincrement, nextvalue from calpontsys.syscolumn  where autoincrement = 'n' and `schema`='mcol5572' and tablename='foo';
```

Reply:
There are an "i_s.test" and "mcol641-create.test" tests in the basic test suite that most probably have the same problem.

Notwithsdanding that, I've changed these SELECT statements per your request.